src/main/resources/hudson/plugins/git/UserMergeOptions/config.jelly

```diff
@@ -13,4 +13,7 @@
   <f:entry title="Merge strategy:" field="mergeStrategy">
       <f:select />
   </f:entry>
-</j:jelly>
\ No newline at end of file
+  <f:entry title="Fast Forward Mode:" field="fastForwardMode">
+    <f:select />
```

Comment:
Styles is not a problem. Try locally change this ui to work with enum, because it looks like right form instead of manually populating ListBox data, etc for select form. As i see for enum you just need bind variable to value.
Some hints how you can search for existed implementations https://wiki.jenkins-ci.org/display/JENKINS/Grepping+all+sources